### DISCUSSION TITLE
Occasionally getting 504 proxy-timeout

### DISCUSSION THREAD

**Discussion by shirabendorskyformation** [community/community#24483] [Apps, API and Webhooks]
<p>I have an application that is installed in different customers site.</p>
<p>Some customers are constantly seeing the error: <strong>Bad response status from link <a href="https://github.ual.com/api/v3/user/repos" rel="nofollow noopener">https://github.ual.com/api/v3/user/repos</a>.</strong></p>
<p><strong>Status is Proxy Timeout ( The connection timed out)</strong></p>
<p>While others and myself don’t see this at all.</p>
<p>What can explain this and how can it be solved?</p>
<p>Thanks</p>
<p>Shira</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You would have to contact the owner of the <a href="http://github.ual.com" rel="nofollow noopener">github.ual.com</a> server for more information on that, since it isn’t a service that github maintains. It appears that <a href="https://whois.domaintools.com/ual.com" rel="nofollow noopener">ual.com</a> is owned by United Airlines. I assume your customer’s site is on their premises? Perhaps their IT department can help with the problem.</p>
